# Definition of ROS service LocGetSystemState for sick localization.
#
# ROS service LocGetSystemStatep queries the error status
# by sending cola command ("sMN LocGetSystemState").
#
# See Telegram-Listing-v1.1.0.241R.pdf for further details about 
# Cola telegrams and this command.

#
# Request (input)
#

---

#
# Response (output)
#

string systemstate # The current system state. Max length = 100
bool success # true: system ready response received from localization controller, false: system not ready or service failed (timeout or error status from controller)

